src/main/java/org/fogbowcloud/manager/core/datastore/orderstorage/OrderTimestampStorage.java

```diff
@@ -0,0 +1,65 @@
+package org.fogbowcloud.manager.core.datastore.orderstorage;
+
+import org.apache.log4j.Logger;
+import org.fogbowcloud.manager.core.datastore.commands.TimestampSQLCommands;
+import org.fogbowcloud.manager.core.models.orders.Order;
+
+import java.sql.*;
+import java.util.Date;
+
```

Comment:
add a comment here why do we need this special *TimestampStorage class. What is the feature we added?

Reply:
Done!